src/rp2_common/pico_lwip/CMakeLists.txt

```diff
@@ -299,7 +299,8 @@ if (EXISTS ${PICO_LWIP_PATH}/${LWIP_TEST_PATH})
     target_link_libraries(pico_lwip_freertos INTERFACE
             pico_async_context_base
             pico_lwip
-            pico_lwip_contrib_freertos)
+            pico_lwip_contrib_freertos
+            pico_rand)
```

Comment:
why pico_rand here?

Reply:
I was getting a build error building pico-examples. But now I look again I'm not sure why... let me recheck.

Reply:
On develop pico_lwip_nosys links to pico_rand. So pico_lwip_freertos needs to link to it as well?

Reply:
yes, sorry, i thought i had fixed this already

Reply:
and indeed committed separately... (and was in my original feedback for pico_rand!)